apps/web/src/pages/_app.tsx

```diff
@@ -11,10 +11,8 @@ function MyApp({ Component, pageProps }: AppProps) {
   return (
     <div className='h-full flex flex-col'>
       <Header />
-      <main className='flex-grow bg-bcLightBackground flex justify-center md:pt-11 pt-5'>
-        <div className='h-min md:w-layout w-full md:mx-0 mx-2 mb-12'>
-          <Component {...pageProps} />
-        </div>
+      <main className='flex-grow flex justify-center'>
```

Comment:
Any particular reason for this change?

Reply:
Designs have different background colors for the form and the confirmation screen so pages need to maintain their own layouts

Reply:
Every form page's background is gray but confirmation's white.